submissions/KelechiPreciousNwachukwu/css/signup.css

```diff
@@ -45,6 +45,14 @@ h2 {
     font-size: 2.5rem;
```

Comment:
if you are using this CSS file for styling both `signup` and `contact` pages, kindly give the name of this file a generic name. the name `signup.css` kind of shows it is just for the `signup` page